samples/Config.CustomConfigurationProviders.Web/project.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "*.config ConfigurationProviders Library",
+  "authors": [ "Microsoft" ],
+  "tags": [ "configuration" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
+  "content": [
+    "Web*.config"
+  ],
+
+  "dependencies": {
+    "Microsoft.Extensions.Configuration": "1.0.0-rc1-final",
```

Comment:
I had it as * before, but `Microsoft.Extensions.Configuration.Constants.KeyDelimiter` doesn't exist in the rc2 packages that are getting restored. :( Is that something that is being removed?